src/index.html

```diff
@@ -48,7 +48,7 @@
     <!-- Use cdn for vite, but this is served from static by Django -->
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.0/font/bootstrap-icons.css"
```

Comment:
This is failing to load from jsdelivr on merge-ci: `(blocked:csp)`.
We actually serve a static copy of this in production. See `views.py`:

```
    # bootstrap-icons. Use CDN when served by vite, but use static copy
    # when served by omero-web
    html = html.replace(
        "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/",
        static_dir)
```
So you'll also need to update the copy under `/static/bootstrap-icons.css` and also update `views.py` accordingly.

Apologies for the confusing steps - any suggestions for improving it are welcome!

Reply:
Thanks Will for spotting my mistake. 
It worked in my dev omero-figure, but good thing that your CI caught it.